.pylintrc

```diff
@@ -172,7 +172,7 @@ attr-rgx=[a-z_][a-z0-9_]{2,30}$
 attr-name-hint=[a-z_][a-z0-9_]{2,30}$
 
 # Regular expression matching correct argument names
-argument-rgx=[a-z_][a-z0-9_]{2,30}$
+argument-rgx=[a-z_][a-z0-9_]{2,30}|ax$
```

Comment:
I think this allows to remove https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/visualization/gate_map.py#L14

Reply:
Yep, it does. Do you want me to do that here? Or do it after merging this?

Reply:
changed in 36613777